addon/components/document-details.hbs

```diff
@@ -110,7 +115,7 @@
             </a>
           {{else}}
             <a
-              class="uk-link-reset"
+              class="uk-link"
```

Comment:
I don't think this class is necessary, UIkit styles links like this per default. `.uk-link` is used to style non link tags like links.

Reply:
Good point.

Reply:
Done